spring-batch-infrastructure/src/main/java/org/springframework/batch/item/database/JpaItemWriter.java

```diff
@@ -91,6 +103,9 @@ public void write(Chunk<? extends T> items) {
 		}
 		doWrite(entityManager, items);
 		entityManager.flush();
+		if (Boolean.TRUE.equals(clearSession)) {
```

Comment:
Why not `if (clearSession)`? It is a primitive boolean, so no need for a more complex if statement IMHO.

Reply:
Since the clearSession variable can't be null, I think it would be better to change it as you suggest. Thanks for the review.